integration-tests/common/src/main/java/com/okta/spring/tests/common/jwt/BasicJwtResourceServerApplication.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Okta, Inc.
+ * Copyright 2021-Present Okta, Inc.
```

Comment:
Don't change the start date on license headers, that's the important part of the copyright